.github/workflows/helm-chart-test.yaml

```diff
@@ -0,0 +1,20 @@
+name: Helm Chart Test
+
+on:
+  pull_request: {}
+
+jobs:
+  test:
+    name: Helm Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run check
+        run: |
+          helm lint helm/weave-gitops
+          kind create cluster
+          helm install --dry-run --debug weave-gitops helm/weave-gitops
+          kind delete cluster
```

Comment:
FYI not having a newline at the end of the file causes diff (and git apply) to warn the user.  Also, if someone were to use >> to append to this file they wouldn't get what they expected.  Do you have strong objections to adding newlines back in?

Reply:
Nope, I'll add it.